theano/configdefaults.py

```diff
@@ -1580,7 +1580,7 @@ def filter_vm_lazy(val):
 
 
 def _timeout_default():
-    return theano.config.compile.wait * 24
+    return theano.config.compile.wait * 48
```

Comment:
why change this? The current number is 5*24, so 120s. What prompted you to change this?